src/api.ts

```diff
@@ -42,6 +42,7 @@ export type CommitHash = string;
 export type BranchName = string;
 export type PortNumber = number;
 export type ImageStatus = 'NoImage' | 'Inactive' | PortNumber;
+export type RunEnv = string;
```

Comment:
Nit: Is this by any chance a defined set of strings? Maybe `'jetpack' | 'calypso'` would be a more descriptive type?

Reply:
@saramarcondes - I didn't want to do that in case of there being additional envs added going forward, updating that in multiple places seemed counter to having the config in general. If there's way to have the type pull that in I'm happy to defer to your expertise! :)

Reply:
Note I merged this in order to let us continue, but I'm happy to open a small PR to fix if there's a good answer here.

Reply:
Oh yup, sorry, I had the review open for too long 😭 I don't think this is too important to spend too much time on 👍